Modules/SubModule_Import/xASL_imp_DCM2NII_ConvertScan.m

```diff
@@ -149,6 +149,15 @@
     end
 
     %% 4.6 Start the conversion if this scan should not be skipped
+    
```

Comment:
Indeed, if everything is done on the subject level, then everything should be managed at the subject level. So we either delete everything for a single subject and rerun everything for this subject, or we don't at all.